jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  testEnvironment: 'node',
+  roots: ['<rootDir>/test'],
+  testMatch: ['**/*.test.ts'],
+  transform: {
+    '^.+\\.tsx?$': 'ts-jest'
+  }
```

Comment:
How would you add tests coverage?

Reply:
Updated with 

`collectCoverageFrom: ['**/*.{ts}', '!**/*.d.ts', '!**/node_modules/**'],`